showcase/src/app/demo-ui/pages/table/demo-table-page.component.html

```diff
@@ -40,5 +40,14 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 	>
 		<showcase-table-with-custom-styling></showcase-table-with-custom-styling>
 	</example-viewer>
+
+	<example-viewer
+		id="withStickyActions"
```

Comment:
Can you move this new table one place above? Just below the other table about the fixed header so wa can see the 2 examples about fixed header/columns together ;)